webcompat/static/js/lib/issues.js

```diff
@@ -484,6 +485,10 @@ issues.MainView = Backbone.View.extend({
   },
   render: function() {
     this.$el.fadeIn();
+  },
+
+  handleKeyShortcuts: function(e) {
+     Mousetrap.bind('mod+enter', this.addNewComment() ,'keydown');
```

Comment:
This is probably a scope/`this` binding issue -- so possible something like:

`Mousetrap.bind('mod+enter', _.bind(this.addNewComment, this), 'keydown');` will work (or similar -- untested!)

Is there any reason we need the 3rd `keydown` arg? AFAIK, Mousetrap will just do the right thing™ without it.

However, thinking more about this, with this approach aren't we re-binding a new listener on every keydown event? That doesn't seem to be what we want. 

We should probably remove the "keydown: handleKeyShortcuts" key/value pair from the `events` object and just call `this.handleKeyShortcuts()` from inside the `initialize` method.